### PR TITLE
Added PCSX2 instructions

### PR DESCRIPTION
**Pull request by Nova791**
Added basic instructions to make PCSX2 work flawlessly on SteamOS in gamemode

### CODE REVIEW

**Comment by nchristopher** [comment]
Could you give me permission to edit this PR so that I can add the initial section to the guide, along with a few changes?

**Comment by Nova791** [comment]
You should be able, I have Allow edits by maintainers toggled. Is it not letting you?

**Comment by nchristopher** [comment]
It wasn't letting me, I'm probably doing something wrong then. I'll just submit my edits as suggestions.

File: README.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ roms
 ├── n64
 ├── snes
 ├── pce
+├── playstation 2
```
Comment (nchristopher) [inline README.md:45]: We shouldn't recommend spaces in directory names, it complicates some things.
```suggestion
├── ps2
```

File: emulators/pcsx2.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  
+Open Advanced settings, and set the following:
+  Precache Textures: toggle ON
+
+## Steam ROM Manager
+
+Open Steam ROM Manager, press Parsers, then enter the following settings:
+
+-   Select `Sony Playstation 2 - PCSX2` under Community Presets.
+-   You can add additional categories to Steam category using the format `${category name}`, this is case-sensitive. Example: ${Playstation 2}
```
Comment (nchristopher) [inline emulators/pcsx2.md:22]: This example isn't necessary, the parser preset already includes `${PS2}`.

```suggestion
-   You can add additional categories to Steam category using the format `${category name}`, this is case-sensitive.
```

File: emulators/pcsx2.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+## Configuring PCSX2
```
Comment (nchristopher) [inline emulators/pcsx2.md:1]:
```suggestion
# PCSX2 Guide

❗ Follow the [Getting Started section](../README.md#getting-started) of `README.md` first.

This section of the guide explains how to setup [PCSX2](https://pcsx2.net/), a PlayStation 2 emulator.

## Installing PCSX2

Open Discover, SteamOS' app store, then search for **PCSX2**, then click on it.

![](https://user-images.githubusercontent.com/58091943/159169454-555ab88e-5bce-4e29-b27d-f18192931e56.png)

In the top right, select **Sources**, then **Flatpak**, then press **Install**.

Alternatively, open up a terminal and run

```bash
flatpak install --user -y net.pcsx2.PCSX2```

## Configuring PCSX2
```

File: emulators/pcsx2.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+## Steam ROM Manager
+
+Open Steam ROM Manager, press Parsers, then enter the following settings:
+
+-   Select `Sony Playstation 2 - PCSX2` under Community Presets.
+-   You can add additional categories to Steam category using the format `${category name}`, this is case-sensitive. Example: ${Playstation 2}
+-   Set executable to `/usr/bin/flatpak`.
+-   Set command line arguments to `run net.pcsx2.PCSX2 "${filepath}" --nogui --fullscreen`
+-   Set ROMs directory to wherever your PS2 ROMs are. This is the same location from before.
```
Comment (nchristopher) [inline emulators/pcsx2.md:25]:
```suggestion
- Set ROMs directory to wherever your PS2 ROMs are - if you're using the recommended path, this should be `~/roms/ps2`.
```

File: emulators/pcsx2.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+OpenConfig -> Graphics Settings, Set the following:
+  Internal Resolution: 2x Native
+  CRC Hack Level: Aggressive
+  
+Open the Hacks tab, and set the following:
+  Manual HW Hacks: toggle ON
+  Fast Texture Invalidation: toggle ON
+  
+Open Advanced settings, and set the following:
+  Precache Textures: toggle ON
```
Comment (nchristopher) [inline emulators/pcsx2.md:15]:
```suggestion
Open up the emulator, then navigate through the initial setup. You'll have to provide your own PS2 BIOS file.

Navigate to **Config > General Settings**, then in the Speedhacks menu, enable MTVU under the microVU Hacks section.

Open GS Window in General Settings, then change the aspect ratio to "Fit to Window/Screen", then apply & close general settings.

Navigate to **Config > Graphics Settings**, then set the following:
- Internal Resolution: 2x Native
- CRC Hack Level: Aggressive
  
Open the Hacks tab, then set the following:
- Manual HW Hacks: toggle ON
- Fast Texture Invalidation: toggle ON
  
Open Advanced settings, then set the following:
- Precache Textures: toggle ON
```

**Review by nchristopher — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -74,6 +75,7 @@ From here, the guide branches off for each system you want to emulate:
 -   [Nintendo 64](./emulators/m64p.md)
 -   [SNES](./emulators/snes9x.md)
 -   [PC Engine (CD) / TurboGrafx-16 (-CD) / SuperGrafx](./emulators/mednaffe.md)
+-   [Playstation 2](./emulators/pcsx2.md)
```
Comment (nchristopher) [inline README.md:78]:
```suggestion
-   [PlayStation 2](./emulators/pcsx2.md)
```

**Review by nchristopher — APPROVED** [review]

**Pull request merged**